### DISCUSSION TITLE
node:1340) MaxListenersExceededWarning: Possible EventTarget memory leak detected. 11 abort listeners added to [AbortSignal]. MaxListeners is 10. Use events.setMaxListeners() to increase limi

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [bbc/sqs-consumer#503] [Help]
### Summary
is sqs-consumer using Event emitters or something ? 
can I use```ts process.setMaxListeners(0);``` to fix it?

![image](https://github.com/bbc/sqs-consumer/assets/35369563/1bf1d915-5be7-49f0-8959-7c15c3f84334)

**Comment by nicholasgriffintn** [comment]
There's an issue for this here: https://github.com/bbc/sqs-consumer/issues/500

Currently, you'll need to pin this dependency:

```
 "@smithy/node-http-handler": "3.0.1",
```
